### PR TITLE
DOC-600: broken links

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/4.7/enterprise/app-upgrade-notes.md [hunk 1]
```diff
@@ -866,7 +866,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.7/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](release-notes.html#jpa-access-in-flows-and-services) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.7/enterprise/app-upgrade-notes.md:869]: This link is still broken. Think it should be release-notes-enterprise. That header doesn't exist though...

File: content/en/platform/corda/4.6/enterprise/app-upgrade-notes.md [hunk 2]
```diff
@@ -861,7 +861,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.6/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](release-notes.html#jpa-access-in-flows-and-services) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.6/enterprise/app-upgrade-notes.md:864]: This link is still broken. Think it should be release-notes-enterprise. That header doesn't exist though...

File: content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md [hunk 3]
```diff
@@ -829,7 +829,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](../../../../../../en/api-ref/corda/4.5/open-source/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](release-notes.html#jpa-access-in-flows-and-services) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md:832]: This link is still broken. Think it should be release-notes-enterprise. That header doesn't exist though...

File: content/en/platform/corda/4.3/enterprise/app-upgrade-notes.md [hunk 4]
```diff
@@ -779,7 +779,7 @@ You will be able to use this feature if the compatibility zone you plan to deplo
 
 to check the correctness of the transaction. Please take this into account for your own schedule planning.You can read more about signature constraints and what they do in [API: Contract Constraints](api-contract-constraints.md). The `TransactionBuilder` class will
 automatically use them if your application JAR is signed. **We recommend all JARs are signed**. To learn how to sign your JAR files, read
-[Signing the CorDapp JAR](cordapp-build-systems.md#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
+[Signing the CorDapp JAR](cordapp-build-systems.html#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```
Comment (michelemarsden) [inline content/en/platform/corda/4.3/enterprise/app-upgrade-notes.md:782]:
```suggestion
[Signing the CorDapp JAR](cordapp-build-systems.html#signing-the-cordapp-jar). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```

File: content/en/platform/corda/4.3/enterprise/app-upgrade-notes.md [hunk 5]
```diff
@@ -841,7 +841,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.3/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](release-notes.html#jpa-access-in-flows-and-services) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.3/enterprise/app-upgrade-notes.md:844]: This link is still broken. Think it should be release-notes-enterprise. That header doesn't exist though...

File: content/en/platform/corda/4.2/enterprise/app-upgrade-notes.md [hunk 6]
```diff
@@ -606,7 +606,7 @@ You will be able to use this feature if the compatibility zone you plan to deplo
 
 to check the correctness of the transaction. Please take this into account for your own schedule planning.You can read more about signature constraints and what they do in [API: Contract Constraints](api-contract-constraints.md). The `TransactionBuilder` class will
 automatically use them if your application JAR is signed. **We recommend all JARs are signed**. To learn how to sign your JAR files, read
-[Signing the CorDapp JAR](cordapp-build-systems.md#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
+[Signing the CorDapp JAR](cordapp-build-systems.html#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```
Comment (michelemarsden) [inline content/en/platform/corda/4.2/enterprise/app-upgrade-notes.md:609]:
```suggestion
[Signing the CorDapp JAR](cordapp-build-systems.html#signing-the-cordapp-jar). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```

File: content/en/platform/corda/4.2/enterprise/app-upgrade-notes.md [hunk 7]
```diff
@@ -668,7 +668,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.2/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](release-notes.html#jpa-access-in-flows-and-services) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.2/enterprise/app-upgrade-notes.md:671]: This link is still broken. Think it should be release-notes-enterprise. That header doesn't exist though...

File: content/en/platform/corda/4.1/enterprise/app-upgrade-notes.md [hunk 8]
```diff
@@ -655,7 +655,7 @@ automatically using new signature constraints selected automatically based on th
 
 You can read more about signature constraints and what they do in [API: Contract Constraints](api-contract-constraints.md). The `TransactionBuilder` class will
 automatically use them if your application JAR is signed. **We recommend all JARs are signed**. To learn how to sign your JAR files, read
-[Signing the CorDapp JAR](cordapp-build-systems.md#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
+[Signing the CorDapp JAR](cordapp-build-systems.html#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```
Comment (michelemarsden) [inline content/en/platform/corda/4.1/enterprise/app-upgrade-notes.md:658]:
```suggestion
[Signing the CorDapp JAR](cordapp-build-systems.html#signing-the-cordapp-jar). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```

File: content/en/platform/corda/4.1/enterprise/app-upgrade-notes.md [hunk 9]
```diff
@@ -715,7 +715,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.1/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](release-notes.html#jpa-access-in-flows-and-services) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.1/enterprise/app-upgrade-notes.md:718]: This link is still broken. Think it should be release-notes-enterprise. That header doesn't exist though...

File: content/en/platform/corda/4.0/enterprise/app-upgrade-notes.md [hunk 10]
```diff
@@ -604,7 +604,7 @@ You will be able to use this feature if the compatibility zone you plan to deplo
 
 to check the correctness of the transaction. Please take this into account for your own schedule planning.You can read more about signature constraints and what they do in [API: Contract Constraints](api-contract-constraints.md). The `TransactionBuilder` class will
 automatically use them if your application JAR is signed. **We recommend all JARs are signed**. To learn how to sign your JAR files, read
-[Signing the CorDapp JAR](cordapp-build-systems.md#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
+[Signing the CorDapp JAR](cordapp-build-systems.html#cordapp-build-system-signing-cordapp-jar-ref). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```
Comment (michelemarsden) [inline content/en/platform/corda/4.0/enterprise/app-upgrade-notes.md:607]:
```suggestion
[Signing the CorDapp JAR](cordapp-build-systems.html#signing-the-cordapp-jar). In dev mode, all JARs are signed by developer certificates. If a JAR that was signed
```

**Review by michelemarsden — COMMENTED** [review]
Some minor corrections.

File: content/en/platform/corda/4.1/enterprise/app-upgrade-notes.md (hunk 9 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.1/enterprise/app-upgrade-notes.md:718]: No such section in rel notes enterprise. Updated with the link to a different section that covers JPA support and the withEntityManager method.

File: content/en/platform/corda/4.2/enterprise/app-upgrade-notes.md (hunk 7 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.2/enterprise/app-upgrade-notes.md:671]: No such section in rel notes enterprise. Updated with the link to a different section that covers JPA support and the withEntityManager method.

File: content/en/platform/corda/4.3/enterprise/app-upgrade-notes.md (hunk 5 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.3/enterprise/app-upgrade-notes.md:844]: No such section in rel notes enterprise. Updated with the link to a different section that covers JPA support and the withEntityManager method.

File: content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md (hunk 3 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md:832]: No such section in rel notes enterprise. Updated with the link to a different section that covers JPA support and the withEntityManager method.

File: content/en/platform/corda/4.7/enterprise/app-upgrade-notes.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.7/enterprise/app-upgrade-notes.md:869]: No such section in rel notes enterprise. Updated with the link to a different section that covers JPA support and the withEntityManager method.

File: content/en/platform/corda/4.6/enterprise/app-upgrade-notes.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.6/enterprise/app-upgrade-notes.md:864]: No such section in rel notes enterprise. Updated with the link to a different section that covers JPA support and the withEntityManager method.

File: content/en/platform/corda/4.7/enterprise/app-upgrade-notes.md [hunk 11]
```diff
@@ -866,7 +866,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.7/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](state-persistence.html#jpa-support) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.7/enterprise/app-upgrade-notes.md:869]: So I think the path should be this. However, I can't locally build to check. 
```suggestion
* The [new withEntityManager API](cordapps/state-persistence.html#jpa-support) for using JPA inside your flows and services.
```

File: content/en/platform/corda/4.6/enterprise/app-upgrade-notes.md [hunk 12]
```diff
@@ -861,7 +861,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](https://api.corda.net/api/corda-os/4.6/html/api/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](state-persistence.html#jpa-support) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.6/enterprise/app-upgrade-notes.md:864]: So I think the path should be this. However, I can't locally build to check. 
```suggestion
* The [new withEntityManager API](cordapps/state-persistence.html#jpa-support) for using JPA inside your flows and services.
```

File: content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md [hunk 13]
```diff
@@ -829,7 +829,7 @@ may result in queries returning more states than expected if the node is using o
 Corda 4 adds several new APIs that help you build applications. Why not explore:
 
 
-* The [new withEntityManager API](../../../../../../en/api-ref/corda/4.5/open-source/javadoc/net/corda/core/node/ServiceHub.html#withEntityManager-block-) for using JPA inside your flows and services.
+* The [new withEntityManager API](state-persistence.html#jpa-support) for using JPA inside your flows and services.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md:832]: So I think the path should be this. However, I can't locally build to check. 
```suggestion
* The [new withEntityManager API](cordapps/state-persistence.html#jpa-support) for using JPA inside your flows and services.
```

**Review by michelemarsden — COMMENTED** [review]
Couple of link corrections.

**Review by michelemarsden — APPROVED** [review]
LGTM

**Pull request merged**